Dispatcher/Constants/APIConstants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+struct APIConstants {
+    static let latestHeadlinesNewscatcherURL = "https://api.newscatcherapi.com/v2/latest_headlines"
+    static let searchArticlesNewscatcherURL = "https://api.newscatcherapi.com/v2/search"
+    static let APIkey = "pxIuyKmcbGplzZfNRXbSQOKltrJ7Zgi4gHsvdWoM43g"
+    static let ARTICLES_TO_FETCH_AMOUNT = 12
```

Comment:
Stick with the same naming convention - don't have camelCase but then also THIS_WAY and also CAMELcase